.idea/.generators

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
You typically want to keep your `.idea/` files out of git, because they change a lot and just add noise.